pkg/kanctl/actionset.go

```diff
@@ -772,6 +779,19 @@ func verifyRepositoryServerParams(repoServer *crv1alpha1.ObjectReference, crCli
 	return nil
 }
 
+func waitForKopiaRepositoryServerReady(ctx context.Context, crCli versioned.Interface, rs *crv1alpha1.RepositoryServer) error {
+	timeoutCtx, waitCancel := context.WithTimeout(ctx, contextWaitTimeout)
+	defer waitCancel()
+	pollErr := poll.Wait(timeoutCtx, func(ctx context.Context) (bool, error) {
+		repositoryServer, err := crCli.CrV1alpha1().RepositoryServers(rs.GetNamespace()).Get(ctx, rs.GetName(), metav1.GetOptions{})
```

Comment:
We should continue checking if the supplied repo server doesn't exist right? Something like
```
		if apierrors.IsNotFound(errors.Cause(err)) {
			return false, nil
		}
```

Reply:
@pavannd1 we're calling this function i.e `waitForKopiaRepositoryServerReady` in `verifyRepositoryServerParams` function and there we're checking if the Repository Server is present or not.

So before polling, we're making sure that RepositoryServer CR exists